tests/external/test_stereoscope.py

```diff
@@ -0,0 +1,26 @@
+import scvi
+import os
+
+import numpy as np
+from scvi.external.stereoscope import RNAStereoscope, SpatialStereoscope
```

Comment:
you shouldn't need `scvi.external.stereoscope` -- it should be just `scvi.external`.